fixing error "style is not done loading"

andre-dietrich opened the pull request:
a little modification according to the following suggestion : 

https://bl.ocks.org/malwoodsantoro/43bba1d03a8d22c10b3e9943eb0c55d1

gampleman commented:
Actually I’ve been investigating this side of mapbox recently and it’s pretty nasty. Thanks for the PR, I’ll need a bit of time to look into it.

andre-dietrich commented:
Hi, I just updated Element and added a zoom and a center attribute function, this functionality was already implemented at the JS part, but not within elm... probably you could also just copy it into your projects, since it is pretty basic ... Regards ...

gampleman commented:
@andre-dietrich those are not implemented on purpose. They form a basis of a "controlled" mode - which would be useful, but there's a lot more required to actually implement it. Also, this can already be achieved using map styles, so I'd rather not have 2 ways to do the same thing.
